com.reprezen.swagedit/src/com/reprezen/swagedit/validation/Validator.java

```diff
@@ -160,6 +177,74 @@ protected boolean hasArrayType(AbstractNode node) {
         return false;
     }
 
+    /**
+     * Validates an object type definition.
+     * 
+     * @param errors
+     * @param node
+     */
+    protected void checkObjectTypeDefinition(Set<SwaggerError> errors, AbstractNode node) {
+        if (node instanceof ObjectNode) {
+            JsonPointer ptr = node.getPointer();
+            if (ptr != null && ptr.toString().startsWith("/definitions")) {
```

Comment:
Yes, inline structures are not validated, e.g. for this model:
```yaml
swagger: '2.0'
info:
  version: 0.0.0
  title: Simple API
paths:
  /foo:
    get:
      summary: Read
      description: Provide details for the entire list (for collection resources) or an item (for object resources)
      responses:
        '200':
          description: OK
          schema:
            properties:
              bar:
                type: string
```
But how can we define if an element is a schema definition or not?...